host/4/bookworm/dotnet/dotnet8.Dockerfile

```diff
@@ -0,0 +1,32 @@
+# Build the runtime from source
+ARG HOST_VERSION=4.834.0
+FROM mcr.microsoft.com/dotnet/sdk:8.0-bookworm-slim-amd64 AS runtime-image
+ARG HOST_VERSION
+
+ENV PublishWithAspNetCoreTargetManifest=false
+
+RUN BUILD_NUMBER=$(echo ${HOST_VERSION} | cut -d'.' -f 3) && \
+    git clone --branch v${HOST_VERSION} https://github.com/Azure/azure-functions-host /src/azure-functions-host && \
+    cd /src/azure-functions-host && \
+    HOST_COMMIT=$(git rev-list -1 HEAD) && \
+    dotnet publish -v q /p:BuildNumber=$BUILD_NUMBER /p:CommitHash=$HOST_COMMIT src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj -c Release --output /azure-functions-host --runtime linux-x64 --framework net8.0 --self-contained /p:MinorVersionPrefix=8 && \
+    mv /azure-functions-host/workers /workers && mkdir /azure-functions-host/workers && \
+    rm -rf /root/.local /root/.nuget /src
```

Comment:
There are also no extension bundles at all in this image now. is that intentional?

Reply:
yeah I don't think we need them since the E2E test works w/o them. Brett was also mentioning to me that we don't need them anymore

Reply:
My e2e tests only test HTTP which would generally not need ext bundles. Unless you have tested these images e2e in Azure I don't think that my tests are sufficient to say that. But if y'all have decided that you  do not need the ext bundles that is fine

Reply:
Yes double checked and this is expected